lib/chef/resource/windows_dns_record.rb

```diff
@@ -42,18 +42,33 @@ class WindowsDnsRecord < Chef::Resource
         description: "The type of record to create, can be either ARecord, CNAME or PTR.",
         default: "ARecord", equal_to: %w{ARecord CNAME PTR}
 
+      property :dns_server, String,
+        description: "The name of the DNS server on which to create the record.",
+        default: "localhost"
```

Comment:
```suggestion
        default: "localhost",
        introcued: "16.3"
```

Reply:
@tas50 I'm assuming that is meant to say "introduced" instead of "introcued"? I haven't found any documentation anywhere on that property attribute from my attempts to search, do you have any? Is it just for documentation purposes to indicate the version when the property was added, or does it serve a technical purpose to not allow the property for any version prior to the one indicated?

Reply:
That was a typo. We use the introduced data in properties to auto generate our docs pages. I added it in a PR after this so this property gets flagged as being in 16.3 and later in the docs.